src/Controller/Component/CrudComponent.php

```diff
@@ -660,6 +660,16 @@ public function entity(array $data = []): EntityInterface
         return $this->model()->newEntity($data);
     }
 
+    /**
+     * Returns new empty entity.
+     *
+     * @return \Cake\Datasource\EntityInterface
+     */
+    public function emptyEntity(): EntityInterface
```

Comment:
I don't understand why do you need this method?

Reply:
I thought this completed/complemented the existing entity($data) method when you have to do one action manually, and therefore need sometimes the former or latter.
One could also manually call the table, but it seems this one has the internal context and knows the correct tables right away.

Reply:
Nothing wrong with directly using the table instance.

I want to decrease the API surface rather than increasing. Just something I haven't got around to. `CrudComponen::entity()` isn't used within the plugin and should be deprecated. `CrudComponent::useModel()` too isn't needed. If one needs a non default/conventional model they can just set `Controller::$defaultTable` and everything under Crud will use that table.

Reply:
I see